types/test/bulk-create.ts

```diff
@@ -0,0 +1,40 @@
+import { DataTypes, Sequelize } from 'sequelize';
```

Comment:
small nit here: can you add some tests for invalid typing and use `// @ts-expect-error comment explaining why the typing is invalid`?